koku/masu/processor/tasks.py

```diff
@@ -234,10 +238,29 @@ def update_summary_tables(schema_name, provider, provider_uuid, start_date, end_
         start_date, end_date = updater.update_daily_tables(start_date, end_date)
         updater.update_summary_tables(start_date, end_date)
     if provider_uuid:
-        chain(
-            update_charge_info.s(schema_name, provider_uuid, start_date, end_date),
-            refresh_materialized_views.si(schema_name, provider, manifest_id),
-        ).apply_async()
+        curr_date = datetime.datetime.now(tz=pytz.UTC)
```

Comment:
We have this DateAccessor in masu that abstracts out the use of datetime. 

It exists here: https://github.com/project-koku/koku/blob/6a9edf9d74bebd54b7785bd672ec6cb62ad7a4d6/koku/masu/external/date_accessor.py#L37

There is also a koku one. Side note: we should consider consolidating both the future as tech debt tackling. The koku one is more fully featured and has convenience methods to get start/end days from this and last month. It is here: 

https://github.com/project-koku/koku/blob/6a9edf9d74bebd54b7785bd672ec6cb62ad7a4d6/koku/api/utils.py#L45

I would use one of those, probably the koku one here since you are doing month math.

Reply:
ahhh, yes these are way better! 👍